.vscode/launch.json

```diff
@@ -18,6 +18,7 @@
 				// Your Github PAT (Personal Access Token) Your Icon > Settings > Developer settings > Personal Access Tokens > Fine-grained tokens (Public Repositories (read-only) should be enough)
 				"DANGER_GITHUB_API_TOKEN": "",
 				"TARGET_BRANCH_NAME": "main",
+				"DEVTOOL_TELEMETRY_TOKEN": "",
```

Comment:
Was this necessary here? I don't think the bundle size analysis needs it to be specified, even empty.

Reply:
But this is very specifically for when we launch the bundle size analysis through VSCode. What issue do you get if this is not here?

Reply:
Removed it. Won't need it! Thx!